### PR TITLE
doc(tutorials): minor typo and syntax fixes

### PR DESCRIPTION
**Pull request by HoloTheDrunk**
## Description
Multiple minor fixes to tutorials.

## Motivation and Context
Better documentation.

### CODE REVIEW

File: docs/tutorials/3DTiles-point-cloud-pnts.md [hunk 1]
```diff
@@ -167,23 +167,23 @@ to threejs documentation for more information on `Object3D`, `Group`, `Points` a
 Then, we just need to pass this callback to the `{@link C3DTilesLayer}` constructor:
 
 ```js
-    onTileContentLoaded: updatePointCloudSize
```
Comment (jailln) [inline docs/tutorials/3DTiles-point-cloud-pnts.md:170]: This is deprecated, we should use `C3DTILES_LAYER_EVENTS.ON_TILE_CONTENT_LOADED` instead now. Could you take this opportunity to update it in the tutorial please?

**Review by jailln — COMMENTED** [review]
Thanks for the changes :)

File: docs/tutorials/3DTiles-point-cloud-pnts.md (hunk 1 above)
Comment (HoloTheDrunk) [reply docs/tutorials/3DTiles-point-cloud-pnts.md:170]: Updated, though I'm not exactly sure what can and can't be linked so I just assumed "missing from left side menu of itowns-project.org" => can't be linked

**Review by jailln — APPROVED** [review]

**Comment by HoloTheDrunk** [comment]
After investigating it seems like the tests either couldn't load the required `examples` files (seems unlikely unless the webpack config was changed recently) or one of the tests timing out simply stopped the entire run and forcefully failed the other tests with incorrect error states.
The codebase is still largely unknown to me, do you have another explanation for the fails?

**Comment by jailln** [comment]
Yes that's it, it has been fixed just now in #2286, you just need to rebase.

**Comment by Desplandis** [comment]
@HoloTheDrunk Seems to be because we were still using pre-geoplateforme URIs for our vector tiles examples.
Seems that #2286 fixes this ! I will relaunch the CI. =)

**Comment by HoloTheDrunk** [comment]
~~:warning: Don't merge this just yet, the use of the `C3DTILES_LAYER_EVENTS.ON_TILE_CONTENT_LOADED` is invalid.~~
Resolved.

**Review by jailln — APPROVED** [review]

**Pull request merged**